analysis_driver/reader/demultiplexing_parsers.py

```diff
@@ -80,3 +80,22 @@ def parse_conversion_stats(xml_file):
                 (lane.get('number'), unknown_barcode.get('sequence'), unknown_barcode.get('count'))
             )
     return all_barcodes_per_lanes, top_unknown_barcodes_per_lanes
+
+
+def parse_seqtk_fqchk_file(fqchk_file, q_threshold):
+    with open(fqchk_file) as open_file:
+        first_line = open_file.readline()
+        header = open_file.readline().split()
+        all_cycles = open_file.readline().split()
+        first_cycle = open_file.readline().split()
+        nb_read = first_cycle[1]
+        nb_base = all_cycles[1]
```

Comment:
Do we only need the first two lines of data in the fqchk file? Can probably reduce the test file in that case